test/controllers/albums_controller_test.rb

```diff
@@ -0,0 +1,61 @@
+require 'test_helper'
+class AlbumsControllerTest < ActionController::TestCase
+test "should get new" do
+  get :new
+  assert_response :success
+end
+
+test "should be able to create" do
+  post_params = {album: {title:"Trouble Man" }}
+  post :create, post_params
+  assert_response :redirect
+end
+
+test 'creating book changes the number of albums' do
+  assert_difference "Album.count", 1 do
+    post :create, {album: {title: "No Way", creator: "Bob Dylon", description:"So good"}}
+      assert_response :redirect
+  end
+end
+test "should get edit" do
+  get :edit, {id: 1}
+  assert_response :success
+end
+
+test "should be able update a album" do
+  patch :update, {id: albums(:one).id, album: {title: "No Way", creator: "Bob Dylon", description:"So good"}}
+  assert_response :redirect
+end
```

Comment:
We could also test that updating an album does _not_ change the number of albums in the database.